src/main/java/br/com/caelum/mog/domains/models/Offer.java

```diff
@@ -1,21 +1,22 @@
 package br.com.caelum.mog.domains.models;
 
-import org.springframework.context.i18n.LocaleContextHolder;
-import org.springframework.util.Assert;
-
 import java.time.LocalDate;
 import java.time.format.TextStyle;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Locale;
 
+import org.springframework.context.i18n.LocaleContextHolder;
+import org.springframework.util.Assert;
+
 public class Offer {
 
     private Customer customer;
     private Responsible responsible;
     private CaelumInfo caelumInfo;
     private List<Course> courses =  new ArrayList<>();
     private LocalDate date;
+    private CareOf careOfName;
```

Comment:
Binamen, você nao acha deixar aqui como careOf apenas, porque quando for chamar, acho que vai ficar estranho com
 ``` offer.getCareOfName().getCareOfName() ``` ?, ou no getter você da o ``` return careOfName.getCareOfName(); ```

Reply:
Faz todo sentido